package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "firebase-functions",
       "version": "3.19.0",
       "license": "MIT",
       "dependencies": {
```

Comment:
I'd just uncheck this file in this PR since we aren't adding any new dependency.

Reply:
Done.